src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -543,6 +543,9 @@
     <None Include="UI\Fonts\Artifakt-Element-Regular.otf">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
```

Comment:
You should be able to find the fonts now in `extern` folder since the changes are now in `master` and this branch

Reply:
I'm struggling to resolve this at run time: 

![image](https://user-images.githubusercontent.com/29973601/129236243-d68b8627-6385-4de9-8d1a-936ca546abdc.png)

@mjkkirschner Note: I've had to add an (open source) version of Courier so that we can keep using a monospace font in the code block editor. The mockup was showing a proportional width font. Having spoken with the UX team, we're using Courier for code blocks based on some Autodesk guidance we found. The original Courier didn't scale well (I think it was pixel based?) so this new version, Courier Prime, is an OS alternative that looks the same.